qa/tasks/deepsea.py

```diff
@@ -750,21 +994,78 @@ def __init__(self, ctx, config):
                 )
         self.log.debug("munged config is {}".format(self.config))
 
+    def __ceph_health_test(self):
+        cmd = 'sudo salt-call wait.until status=HEALTH_OK timeout=900 check=1'
+        if self.quiet_salt:
+            cmd += ' 2> /dev/null'
+        self.master_remote.run(args=cmd)
+
+    def __check_salt_api_service(self):
+        base_cmd = 'sudo systemctl status --full --lines={} {}.service'
+        try:
+            self.master_remote.run(args=base_cmd.format('0', 'salt-api'))
+        except CommandFailedError:
+            self.master_remote.run(args=base_cmd.format('100', 'salt-api'))
+            raise
+        self.scripts.salt_api_test()
+
+    def __is_stage_between_0_and_5(self):
+        """
+        This is implemented as a separate function because the stage specified
+        in the YAML might be a number or a string, and we really don't care
+        what Python sees it as.
+        """
+        num = self.stage
+        try:
+            num = int(num)
+        except ValueError:
+            return False
+        if num < 0 or num > 5:
+            return False
+        return True
+
+    def __lvm_status(self):
```

Comment:
I would rename such kind of methods to be prefix with 'dump_' because they do not return value but just dumps something.

Reply:
done